src/styles/App.css

```diff
@@ -0,0 +1,49 @@
+.weather-app.light {
+  background-image: url("../../images/daytime_sky.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: top;
+  padding: 2rem;
+  width: 42%;
+  min-height: 40vh;
+  margin: 2rem auto;
+  border-radius: 5rem;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+  font-size: calc(7px + 1vmin);
+  overflow: hidden;
+  box-shadow: rgba(0, 0, 0, 0.15) 0px 10px 10px 0px;
+  color: black;
+}
+
+.weather-app.dark {
+  background-image: url("../../images/night_sky.jpg");
```

Comment:
You are using pretty much all the same properties as above here, so you could combine these classes for all common properties and just declare the things that make them unique in their own declarations, eg:
```
.weather-app.light,
.weather-app.dark {
  // common properties here
}

.weather-app.light {
  // unique properties here
}

.weather-app.dark {
  // unique properties here
}
```

Gets rid of a lot of duplication